common/config/api.go

```diff
@@ -100,6 +105,7 @@ type APIConfig interface {
 	DynamoJobTable() string
 	DynamoTagTable() string
 	LogGroupName() string
+	JobExpiry() int
```

Comment:
Just a thought - but because this is an int, it is unclear if it is minutes, seconds or hours.

Reply:
I'm not sure I can change it, since it can be changed right now via a flag. I'll look into it.

Reply:
I could change it to Duration.

Reply:
I just meant JobExpiry variable name doesn't have any indication of of what time unit. Maybe JobExpiryInSeconds or a comment saying seconds. I'll update this to a `nit` so it doesn't require any changing. I don't know what change I would make to this myself.

Reply:
I made some revisions, and made it a durationflag